index.html

```diff
@@ -1910,39 +1919,44 @@ <h3>Add Proof</h3>
       <section>
         <h3>Add Proof Set/Chain</h3>
         <p>
-          The following algorithm specifies how to incrementally add a proof to
+          The following algorithm specifies how to incrementally add a proof to 
           a proof set or proof chain starting with a secured document containing
           either a proof or proof set/chain.
-          Required inputs are a <a>secured data document</a>
-          (<var>securedDocument</var>) and <a>proof options</a>
+          Required inputs are a <a>secured data document</a> 
+          (<var>securedDocument</var>) and <a>proof options</a> 
           (<var>options</var>). The <a>proof options</a>
-          (<var>options</var>) must satisfy the criteria in
+          (<var>options</var>) must satisfy the criteria in 
           section <a href="#add-proof"></a>.
-          A new <a>secured data document</a> is produced as
-          output. Whenever this algorithm encodes strings, it MUST use UTF-8
+          A new <dfn>secured data document</dfn> is produced as
+          output. Whenever this algorithm encodes strings, it MUST use UTF-8 
           encoding.
```

Comment:
@Wind4Greg, please set your editor to "strip trailing whitespace". This PR is re-introducing trailing spacing characters that were removed in another PR. We do not use trailing whitespace on W3C VCWG specifications.